extensions/rescuetime-focus-session-trigger/src/startFocusTime.ts

```diff
@@ -0,0 +1,47 @@
+import { getPreferenceValues, showToast, Toast } from "@raycast/api";
+import got from "got";
+
+// TODO: implement changelog.md file?
```

Comment:
Should be correct when changing format

Reply:
just missed removing these comments from developing.